README.md

```diff
@@ -3,6 +3,14 @@ KEVM: Semantics of EVM in K
 
 In this repository we provide a model of the EVM in K.
 
+Fast Installation
+-----------------
+
+-   `bash <(curl https://kframework.org/install)`: install [the `kup` utility](https://github.com/runtimeverification/kup)
```

Comment:
Should this list a quick link to the prereq nix installation page for a quick install. I find the single use install is the quickest and least complicated to get working. I have had some challenges with the global install method.